circle-mlir/circle-mlir/tools-test/onnx2circle-value-test/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ endmacro(COPY_SCRIPT)
 
 # copy test scripts
 COPY_SCRIPT(run_value_test.sh)
+COPY_SCRIPT(exec_onnx.py)
```

Comment:
`exec_onnx` is to run ONNX model with random inputs
and save that random inputs and execution outputs from onnx-runtime
to H5 format file.